tools/circleci/xenial-images/Dockerfile

```diff
@@ -93,7 +93,7 @@ RUN apt-get update \
     hunspell-en-us supervisor libssl-dev yui-compressor puppet \
     gettext libffi-dev libfreetype6-dev libz-dev libjpeg-dev \
     libldap2-dev libmemcached-dev python-dev python-pip \
-    python-virtualenv python-six libxml2-dev libxslt1-dev libpq-dev \
+    virtualenv python-six libxml2-dev libxslt1-dev libpq-dev \
```

Comment:
Both python-virtualenv and virtualenv are available in Xenial, https://packages.ubuntu.com/xenial/python-virtualenv. But it looks like installing the latter makes it unnecessary to explicitly install the former